docs/README.md

```diff
@@ -22,6 +22,7 @@ Then build the web site using Quarto:
 
 ```bash
 make site
+# When this completes, the docs will be available at _site/index.html.
```

Comment:
I think our intention was to cover this with `make serve`, which is the best way to preview the website (it's really `quarto preview` under the hood).

I'm not opposed to this note here, and it's unlikely to change in the near term, but it'd be very likely we miss updating this comment if we were to change the docs location in the future for some reason.

Reply:
You're right, and it's too not hard to figure out, regardless. Will drop.